src/Storage/File.php

```diff
@@ -118,11 +118,17 @@ public function getExtension():string {
 		 * @return string
 		 */
 		public function getFileType():string {
-			if (function_exists('finfo_file')) {
-				$info = new \finfo(FILEINFO_MIME_TYPE);
+			static $has_fileinfo;
```

Comment:
Keeping in consistency with the rest of the project, `$has_fileinfo` should be `$hasFileInfo`, defined as a static on the `HTTPContext` class, and referred to as `self::$hasFileInfo`.

Reply:
Fixed